lms/static/js/revisions/views/revision_list_view.js

```diff
@@ -0,0 +1,58 @@
+;(function (define) {
+
+define([
+    'jquery',
+    'underscore',
+    'backbone',
+    'gettext',
+], function ($, _, Backbone, gettext) {
+
+   'use strict';
+
+    return Backbone.View.extend({
+
+        el: '#dashboard-revisions',
+        resultsTemplateId: '#dashboard_revision_results-tpl',
+        resultsItemTemplateId: '#dashboard_revision_item-tpl',
```

Comment:
@bdero I'd suggest renaming these as well to avoid the `results` terminology; how about `revisionListTemplateId` and `revisionItemTemplateId`?